use float type for zoom

caramelli opened the pull request:
fbpdf was used for my presentation on the Linux Framebuffer at FOSDEM 2020 (https://fosdem.org/2020/schedule/event/fbdev).
I replaced the "int" type by "float" for the zoom variable in order to fill the entire screen when using 'f' or 'w' command.
Thanks for this great tool !

aligrudi commented:
Nicolas Caramelli <notifications@github.com> wrote:
> fbpdf was used for my presentation on the Linux Framebuffer at FOSDEM 2020 (https://fosdem.org/2020/schedule/event/fbdev).
> I replaced the &quot;int&quot; type by &quot;float&quot; for the zoom variable in order to fill the entire screen when using &#39;f&#39; or &#39;w&#39; command.
> Thanks for this great tool !

I have seen the slides before by accident and enjoined it a lot.
Thanks for the presentation.

Personally I prefer not to use floating point types, unless
quite necessary; and if zoom is float, zoom_def and mark_row
have to be also.  Instead, I changed them to store zoom level
in percentage.  Please give it a try.

	Ali

caramelli commented:
Perfect!

Nicolas Caramelli

Le dim. 14 juin 2020 à 13:29, Ali Gholami Rudi <notifications@github.com> a
écrit :

> Nicolas Caramelli <notifications@github.com> wrote:
> > fbpdf was used for my presentation on the Linux Framebuffer at FOSDEM
> 2020 (https://fosdem.org/2020/schedule/event/fbdev).
> > I replaced the &quot;int&quot; type by &quot;float&quot; for the zoom
> variable in order to fill the entire screen when using &#39;f&#39; or
> &#39;w&#39; command.
> > Thanks for this great tool !
>
> I have seen the slides before by accident and enjoined it a lot.
> Thanks for the presentation.
>
> Personally I prefer not to use floating point types, unless
> quite necessary; and if zoom is float, zoom_def and mark_row
> have to be also. Instead, I changed them to store zoom level
> in percentage. Please give it a try.
>
> Ali
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/aligrudi/fbpdf/pull/8#issuecomment-643753753>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACEJB6CEALYECVDA3TTEN33RWSYDJANCNFSM4N4B52NQ>
> .
>
